plugins/MultiSites/vue/src/AllWebsitesDashboard/SitesTableSite.vue

```diff
@@ -85,8 +95,8 @@ export default defineComponent({
       type: Object,
       required: true,
     },
-    sparklineDate: String,
     sparklineMetric: String,
+    displaySparkline: Boolean,
```

Comment:
This needs an additional check for `site.isGroup`. Otherwise it may try to fetch an evolution image for group rows and fail because no idSite will be passed.